### ISSUE TITLE
Feature: Undo

### ISSUE THREAD

**Issue by romanofski** [romanofski/purebred#2]
At this point in time this feature is very vague in my understanding. Wishful thinking: I can undo whatever actions I've applied on my mails (except actions like sending ... ). 

For example if I have tagged messages wrong, how can I find the wrong tagged mail again?

**Comment by romanofski** [comment]
We could probably implement a simple undo feature only regarding tags, which stores a list of tags and the operation in a list. Anything more fancy could be something for the future.

**Comment by frasertweedale** [comment]
IMO we should just close this for now.  What is the value?  What cannot be easily undone / repaired with what we have now?  IMO it is unneeded complexity when we have so much else to do.

We can always revisit later but for now I feel that this is just noise.

**Comment by romanofski** [comment]
The value is, that you tag with a shortcut and you can undo your last tagging option effectively making the tagged/stashed away mail or thread re-appear again. If you've fat-fingered the mail away it may take you a while to figure with what you've tagged the mail and find it.

**Comment by frasertweedale** [comment]
Fair enough.  Maybe we could/should make a log of (recent) actions?  IMO this is all pretty low prio stuff tho.

**Comment by romanofski** [comment]
Yeah I like the idea and agree for the low priority. Hence no 1.0 milestone on the bug.

**Comment by frasertweedale** [comment]
@romanofski is this idea still important enough for you to keep this open in the backlog?

**Comment by romanofski** [comment]
Yeah I still like it and want it. I'm pretty sure I'd file it again if we close this one here :)
